tracer/dependabot/Datadog.Dependabot.Integrations.csproj

```diff
@@ -13,8 +13,8 @@
 
     <!-- Integration: Microsoft.Data.Sqlite -->
     <!-- Assembly: Microsoft.Data.Sqlite -->
-    <!-- Latest package https://www.nuget.org/packages/Microsoft.Data.Sqlite/5.0.11 -->
-    <PackageReference Include="Microsoft.Data.Sqlite" Version="5.0.11" />
+    <!-- Latest package https://www.nuget.org/packages/Microsoft.Data.Sqlite/6.0.0 -->
+    <PackageReference Include="Microsoft.Data.Sqlite" Version="6.0.0" />
```

Comment:
Did we capture this major version bump in the integration

Reply:
Unexpected change. 
Plus, after a discussion with Kevin, I understand that we don't proactively support new majors.
I will resolve all related comments

Reply:
These changes are expected aren't they? This isn't indicating any actual change in the _supported_ integrations, but rather the integrations that exist? 

If we don't expect this to change as part of a version bump, then we should remove `DependabotFileManager.UpdateIntegrations()` from the `UpdateIntegrationsJson` Nuke target